examples/p4est_2d_dgsem/elixir_navierstokes_vortex_street.jl

```diff
@@ -101,10 +101,15 @@ analysis_callback = AnalysisCallback(semi, interval = analysis_interval)
 
 alive_callback = AliveCallback(analysis_interval = analysis_interval)
 
+node_variables = Dict{Symbol, Any}()
+# Add `:vorticity` key to `node_variables` dictionary.
+# The actual values are then computed during the `SaveSolutionCallback`.
+node_variables[:vorticity] = nothing
```

Comment:
I wonder whether this is the best interface. It feels a bit odd to create a dictionary and set the entry to `nothing`. One alternative would be to only pass a list of symbols, which can be used internally to create the dict. This is maybe a bit similar to `extra_analysis_errors` from the `AnalysisCallback` (In this case, I would probably rename the argument to something like `extra_node_variables`). Another alternative, which would allow for even more flexibility is to pass a list of functions with a certain signature. These could either we user-defined or pre-defined like a function computing the vorticity. I am not sure, however, if this approach is feasible in general. It would be a bit similar to `extra_analysis_integrals` from the `AnalysisCallback`.

Reply:
Agreed, the dict should be created internally.

Hm passing in functions would be nice - I think it should be doable. 

A more fundamental design choice is probably that we would want another type of solution variables, which truly depend on the solution vector `u_ode` (the subcell coefficients do not directly). This comes with more changes in the implementation, but is the much cleaner way I think.